src/node_postmortem_metadata.cc

```diff
@@ -58,54 +58,78 @@ int GenDebugSymbol();
 #include "util-inl.h"
 #include "req_wrap.h"
 
-// These are the constants describing Node internal structures. Every constant
-// should use the following format:
-//
-//    nodedbg_const_CLASS__MEMBER      Describes a class member constant
-//    nodedbg_offset_CLASS__MEMBER     Describes the offset to a class member
-//
+#define NODEDBG_SYMBOL(Name)  nodedbg_ ## Name
+
+// nodedbg_const_CLASS__MEMBER__TYPE: Describes a class member constant
+#define NODEDBG_CONST(Class, Member, Type) \
+    NODEDBG_SYMBOL(const_ ## Class ## __ ## Member ## __ ## Type)
+
+// nodedbg_offset_CLASS__MEMBER__TYPE: Describes the offset to a class member
+#define NODEDBG_OFFSET(Class, Member, Type) \
+    NODEDBG_SYMBOL(offset_ ## Class ## __ ## Member ## __ ## Type)
+
+// These are the constants describing Node internal structures. The format used
+// for naming them is enforced by the macros above.
+
 // These constants are declared as global integers so that they'll be present in
 // the generated node binary. They also need to be declared outside any
 // namespace to avoid C++ name-mangling.
-uintptr_t nodedbg_offset_BaseObject__persistent_handle_;
 
-int nodedbg_const_Environment__kContextEmbedderDataIndex;
-uintptr_t nodedbg_offset_Environment__handle_wrap_queue_;
-uintptr_t nodedbg_offset_Environment__req_wrap_queue_;
+#define NODE_CONST_POSTMORTEM_METADATA(V)                                     \
+  V(Environment, kContextEmbedderDataIndex, int)
```

Comment:
If there's only one, I'd write it out in full - that's going to be shorter and clearer than doing it through x-macros.